frappe/database/database.py

```diff
@@ -423,14 +423,15 @@ def get_values(self, doctype, filters=None, fieldname="name", ignore=None, as_di
 
 		if isinstance(filters, list):
 			out = self._get_value_for_many_names(
-				doctype,
-				filters,
-				fieldname,
-				order_by,
+				doctype=doctype,
+				names=filters,
```

Comment:
Converted all kwargs since this is quite error-prone. Also, arg names and arg ordering are quite inconsistent between ORM functions 😭